content/source/docs/cloud/sentinel/examples.html.md

```diff
@@ -39,3 +38,5 @@ This page lists some example Sentinel policies. These examples are not exhaustiv
 * [Restrict virtual disk size](https://github.com/hashicorp/terraform-guides/blob/master/governance/second-generation/vmware/restrict-vm-disk-size.sentinel)
 * [Restrict VM CPU count and memory](https://github.com/hashicorp/terraform-guides/blob/master/governance/second-generation/vmware/restrict-vm-cpu-and-memory.sentinel)
 * [Enforce NFS 4.1 and Kerberos](https://github.com/hashicorp/terraform-guides/blob/master/governance/first-generation/vmware/require_nfs41_and_kerberos.sentinel)
+
+-> **Note:** We've developed a number of first-class foundational policies to work out-of-the-box with Amazon Web Services, Microsoft Azure and Google Cloud Platform. Check out the library of [Terraform Foundational Policies](https://github.com/hashicorp/terraform-foundational-policies-library) written by HashiCorp to get up and running with your next [Policy Set](https://www.terraform.io/docs/cloud/sentinel/manage-policies.html#policies-and-policy-sets).
```

Comment:
Should this be higher?

Reply:
@dylanegan I had considered it but there are already so many _note_ and _important_ quotations at the top that I figured the reader would be exhausted by the time they go to the foundational policies section. I would be keen to know if @nfagerlund has any thoughts.